Corzbank.Services/UserService.cs

```diff
@@ -1,15 +1,80 @@
-﻿using Corzbank.Services.Interfaces;
+﻿using AutoMapper;
+using Corzbank.Data.Entities;
+using Corzbank.Data.Entities.Models;
+using Corzbank.Helpers.Validations;
+using Corzbank.Services.Interfaces;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
 using System;
 using System.Collections.Generic;
 using System.Text;
+using System.Threading.Tasks;
 
 namespace Corzbank.Services
 {
     public class UserService : IUserService
     {
-        public void Test()
+        private readonly IMapper _mapper;
+        private readonly UserManager<User> _userManager;
+        private readonly ValidateUser _validateUser;
+        public UserService(IMapper mapper, UserManager<User> userManager, ValidateUser validateUser)
```

Comment:
add an empty line before the constructor

Reply:
resolved